subsys/caf/modules/shell.c

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) 2023 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include <zephyr/shell/shell.h>
+#include <app_event_manager.h>
+#include <caf/events/button_event.h>
+
+static int set_button_handler(const struct shell *shell, size_t argc, char **argv)
+{
+	if (argc < 3) {
+		shell_error(shell, "Missing required parameter(s)");
```

Comment:
where is it documented what the required parameters are?

Reply:
will add some comments about those, thanks.

Reply:
Can you also add some kind of help command / help messaging? Check out some of the other existing shells for examples of how to do this